cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -348,6 +349,7 @@ static void sc_mount_nvidia_driver_biarch(const char *rootfs_dir)
 struct sc_nvidia_driver {
```

Comment:
I think we could add the glcore library pattern to be in the struct, so that cosumer of the structure does not need to guess:

```
struct sc_nvidia_driver {
	int major_version;
	int minor_version;
	int micro_version;
	const char *glcore_pattern;
};
```

Reply:
ok